evasquez26/angular4-startup
# S6-ONE-244 Bug: Error navigating to roles page

## Story / Task

see [ONE-244](https://bixlabs.atlassian.net/browse/ONE-244)


## What this PR does?

Fixes bug: When trying to navigate to roles page and the user doesn't have privileges assigned. ie User: Hamoye Admin. NOTE: We're still missing a fix when the organization is not found

### Before 

It throws error accesing the roles page

### After

No error when accesing roles page

### How to test / reproduce

- Login to the app with user admin@hamoye.com
- Go to roles page (My profile > roles)
- Check no error thrown ('TypeError: Cannot read property 'forEach' of undefined' )

### TODOs

### Related PRs
N/A
## Migrations
NO

Thank you
